pythonsrc/parser/main.py

```diff
@@ -3,6 +3,11 @@
 from sqlglot.optimizer import optimize, qualify
 from sqlglot.lineage import Node
 
+def extract_query(parsed):
+	cols = []
+	for expr in find_all_in_scope(parsed, exp.Column):
+		cols.append(expr)
+	return cols
```

Comment:
why don't we use/improve `extract_columns` instead of this new function? also, the name is not very clear, wdyt?